app/assets/javascripts/application.js

```diff
@@ -14,21 +14,16 @@
 //= require jquery3
 //= require popper
 //= require bootstrap-sprockets
-//= require bs-custom-file-input.js
-//= require bs-file-input-init.js
-//= require react
-//= require react_ujs
 //= require lodash
-//= require components
 //= require tablesort
 //= require tablesort/dist/sorts/tablesort.number.min
-//= require teleconsultation-fields
-//= require_tree .
+//= require bs-custom-file-input
+//= require_tree ./common
 
 $(function () {
+  // initialize tooltips via bootstrap (uses popper underneath)
   $('[data-toggle="tooltip"]').tooltip()
 
-  if($('#analytics-table').length) {
```

Comment:
Moved this to its own JS file.

Reply:
I've kept stuff that is a pure third-party library initialization here, but our own custom initializers in their own files. Open to a different convention as well.

Reply:
Oh I see. Meh, I'm not too worried about it, sounds good.